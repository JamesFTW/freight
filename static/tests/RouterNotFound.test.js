import React from 'react';
import renderer from 'react-test-renderer';
import RouteNotFound from '../components/RouteNotFound.jsx';


test('RouteNotFound Snapshot', () => {
  const component = renderer.create(
    <RouteNotFound />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
