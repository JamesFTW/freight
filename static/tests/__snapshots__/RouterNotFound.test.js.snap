// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RouteNotFound Snapshot 1`] = `
<div>
  <h1>
    Page Not Found
  </h1>
  <p
    className="alert-message notice"
  >
    The page you are looking for was not found.
  </p>
</div>
`;
